kfs-ld/src/main/java/edu/arizona/kfs/module/ld/LaborConstants.java

```diff
@@ -25,4 +25,7 @@
 	
 	public static final String[] BALANCE_TYPES = {"AC", "CB", "IE"};
 
+	//SalaryExpenseTransferDocument
+	public static final String BALANCE_TYPE_ACTUAL = "AC";
+	public static final String BALANCE_TYPE_A2 = "A2";
```

Comment:
Mea culpa, I didn't realize that A2  balance type code was not available on the Systems Options table. I am now OK with the use of these two balance type code constants. As we need to remove your inclusion of OriginEntryFull BO in SalaryExpenseTransferDocument.java anyway, lets go ahead and revert to using the constants.